docs/examples/frameworks/jax/jax-multigpu_example.ipynb

```diff
@@ -27,7 +27,7 @@
   },
```

Comment:
<blockquote> (single GPU training example)[jax-basic_example.ipynb]</blockquote><p>Is it some kind of markup for Sphinx or sth?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/4973/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Fixed</p>